roles/ocp_on_libvirt/defaults/main.yml

```diff
@@ -1,9 +1,23 @@
 ---
 libvirt_image_path: /var/lib/libvirt/images
-enable_conserver: false
```

Comment:
Any reason why now conserver default changed?

Reply:
conserver default state should be enabled so DCI can pull the VMs console outputs and pushed the data to the DCI job as another log file.

I disabled it temporarily so I could have access to the console and debug some issues with the boot mode.

Reply:
Any update?

Reply:
OK,
So I've been reviewing the history and it's true, the role's default value for enable_conserver should be false, as it currently is in the main branch.
In some test environments we have this enabled from the inventory so DCI can pull the VM console outputs.
I'll restore the original value to the variable.